src/winetricks

```diff
@@ -6912,14 +6912,14 @@ helper_dxvk()
         w_try_unzip "${W_TMP}" "${W_CACHE}/${W_PACKAGE}/${_W_package_archive}"
     else
         w_try_cd "${W_TMP}"
-        w_try tar -zxf "${W_CACHE}/${W_PACKAGE}/${_W_package_archive}"
+        w_try tar -zxf "${W_CACHE}/${W_PACKAGE}/${_W_package_archive}" --one-top-level
     fi
 
     for _W_dll in ${_W_dll_overrides}; do
         w_try mv "${W_TMP}/${_W_package_dir}/x32/${_W_dll}.dll" "${W_SYSTEM32_DLLS}/"
         [ "${_W_dll}" = "d3d9" ] && _W_d3d9_support="1"
     done
-    if test "${W_ARCH}" = "win64"; then
+    if test "${W_ARCH}" = "win64" && -d "${W_TMP}/${_W_package_dir}/x64"; then
```

Comment:
Direct3D 8 is only for 32bit applications, so the archive only has a `x32` directory, hence this check.